var/spack/repos/builtin/packages/pbspro/package.py

```diff
@@ -0,0 +1,91 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import llnl.util.tty as tty
+from spack import *
+
+
+class Pbspro(AutotoolsPackage):
+    """PBS Professional software optimizes job scheduling and workload
+    management in high-performance computing (HPC) environments - clusters,
+    clouds, and supercomputers - improving system efficiency and people's
+    productivity."""
+
+    homepage = "https://www.pbspro.org"
+    url = "https://github.com/PBSPro/pbspro/archive/v19.1.3.tar.gz"
+
```

Comment:
The open source licenseed version of pbspro will be renamed OpenPBS.
http://community.pbspro.org/t/openpbs-and-version-20-0/2075
https://github.com/PBSPro/pbspro/ was renamed https://github.com/openpbs/openpbs
(Currently old URL is redirected to new URL.)

Reply:
As far as I understand, they have renamed the repo a very short time ago and the whole renaming is not finished yet. For example, http://openpbs.org is not available yet. Additionally, there will be changes to the building process in version `20.0`. I would keep this PR as it is now and leave the handling of the next renamed version to a person interested in the package. Probably, this will also imply the renaming of the package in Spack from `pbspro` to `openpbs`.

Reply:
Renaming packages can cause inconveniences. We still do it, but if we know the package is going to be renamed in the near future, I think it would be better to be proactive about this, especially for new packages.

Reply:
@adamjstewart OK, I've renamed the package